skyfield/iokit.py

```diff
@@ -184,10 +188,14 @@ def _log(self, message, *args):
     def tle(self, url, reload=False):
         """Load and parse a satellite TLE file.
 
-        Given a URL or a local path, this loads a file of three-line
-        records in the common Celestrak file format, where each first
-        line gives the name of a satellite and the following two lines
-        are the TLE orbital elements.
+        Given a URL or a local path, this loads a file of two- or three-line
+        records in the common Celestrak file format. For a three-line element
```

Comment:
We'll want to maybe remove "Celestrak" here if we're accepting 2-line files, since Celestrak always uses 3 lines.

Reply:
I will reword for my next commit.